### PR TITLE
Mocked Complete Case form

### PR DESCRIPTION
**Pull request by remcohoff**

### CODE REVIEW

File: src/__mocked__/apiSchema.d.ts [hunk 1]
```diff
@@ -3,7 +3,23 @@ declare namespace MockComponents {
 
     export type CompleteCase = {
       readonly id: number
-      readonly title: string
+      readonly explanation: string
```
Comment (gilleswittenberg) [inline src/__mocked__/apiSchema.d.ts:6]: This is now being generated, right?

File: src/app/components/case/forms/CaseCompleteForm/CaseCompleteForm.tsx [hunk 2]
```diff
@@ -11,18 +11,33 @@ type Props = {
...
+    result: data.result?.id ?? null
+  }
+)
+
 const CaseCompleteForm: React.FC<Props> = ({ id, camundaTaskId }) => {
 
-  const [completeCases] = useCompleteCases()
-  const [, { execPost }] = useCompleteCase()
-  const fields = useScaffoldedFields(scaffold, id, completeCases)
+  const themeId = useCase(id)[0]?.theme.id
```
Comment (gilleswittenberg) [inline src/app/components/case/forms/CaseCompleteForm/CaseCompleteForm.tsx:28]: Would prefer:
```
const [caseItem] = useCase(id)
const themeId = caseItem?.theme.id
```

File: src/app/state/rest/case.ts [hunk 3]
```diff
@@ -102,19 +102,56 @@ export const useCompleteCases = (options?: Options) => {
...
-    url: "completeCases",
+    url: "correspondence",
     groupName: "cases",
     handleError,
     isProtected: true,
     isMocked: true
   })
 }
 
+// TODO-MOCKED replace with real endpoint POST
```
Comment (gilleswittenberg) [inline src/app/state/rest/case.ts:118]: Is not mocked anymore, right?

**Review by gilleswittenberg — APPROVED** [review]

File: src/__mocked__/apiSchema.d.ts (hunk 1 above)
Comment (remcohoff) [reply src/__mocked__/apiSchema.d.ts]: Removed it

File: src/app/components/case/forms/CaseCompleteForm/CaseCompleteForm.tsx (hunk 2 above)
Comment (remcohoff) [reply src/app/components/case/forms/CaseCompleteForm/CaseCompleteForm.tsx]: Fixed it

File: src/app/state/rest/case.ts (hunk 3 above)
Comment (remcohoff) [reply src/app/state/rest/case.ts]: Removed it

**Pull request merged**